$NetBSD: patch-af,v 1.3 2011/07/09 21:15:48 asau Exp $

Don't use obsolete header file, contemporary operating systems
(e.g. of BSD family) choose to break it.

--- H/dlmalloc.h.orig	2011-06-11 19:38:36.000000000 +0400
+++ H/dlmalloc.h	2011-07-10 00:22:57.000000000 +0400
@@ -493,14 +493,6 @@
   HAVE_USR_INCLUDE_MALLOC_H.
 */
 
-#if HAVE_MALLOC_H && !defined(_WIN32) && !defined(__NetBSD_Version__)
-#define HAVE_USR_INCLUDE_MALLOC_H 1
-#endif
-
-#ifdef HAVE_USR_INCLUDE_MALLOC_H
-#include <malloc.h>
-#else
-
 /* SVID2/XPG mallinfo structure */
 
 struct mallinfo {
@@ -523,7 +515,6 @@
   so setting them has no effect. But this malloc also supports other
   options in mallopt described below.
 */
-#endif
 
 
 /* ---------- description of public routines ------------ */
